src/Server/Program.fs

```diff
@@ -14,6 +20,23 @@ let getPortsOrDefault defaultVal =
     | null -> defaultVal
     | value -> value |> uint16
```

Comment:
while you at it,
```fsharp
 let GetEnvVar var = 
     match Environment.GetEnvironmentVariable(var) with
     | null -> None
     | value -> Some value
```
=>
```fsharp
 let GetEnvVar var = Environment.GetEnvironmentVariable var |> Option.ofObj
```

Reply:
Actually it'd be nice to have some CLI arguments too. Right now the program gets the config by a mix of CLI args, checking env vars and guessing ;) We have some CLI arguments for host, port, root content and client path with defaults although it's quite primitive. We probably should use Argu :+1: